test/var/runtests.jl

```diff
@@ -86,12 +116,22 @@ end
     @test round(distance(JukesCantor69, dnas2)[2][1], 3) == 1
     @test round(distance(JukesCantor69, m2)[1][1], 3) == 1.648
     @test round(distance(JukesCantor69, m2)[2][1], 3) == 1
+    @test_throws DomainError distance(JukesCantor69, dnas2, 5, 5)
+    d = distance(JukesCantor69, dnas3, 5, 5)
+    a = [0.232616, 0.571605, 0.44084, 0.571605]
+    v = [0.0595041, 0.220408, 0.24, 0.220408]
```

Comment:
a and v are the expected distances and variances for the test cases, according to the JukesCantor69 model. These can be worked out for a given p-distance and sequence length using their formulae using a calculator. Which is what I did here using the JC69 equations from one of my textbooks. So I had did the calculations to get the answers in a and v, and test the function return the right answer. Testing with approx because of floating point and rounding issues.